consumer/dragonboard/dragonboard845c/installation/board-recovery.md

```diff
@@ -65,8 +65,10 @@ UFS provisioning is not support yet.
 
 If the flashing process succeeded, all the right bootloaders and partition table should have been set. And fastboot can now be used to flash Linux root file system. The first thing to try is to get into fastboot, to make sure the flashing completed properly.
 
-* Power off the board and make sure no USB cable is plugged into the board
-* Power on the device using the proper power supply
+* Disconnect the power cable from the board and make sure no USB cable is plugged into the board
+* Hold down the "VOL-" button while reconnecting the power supply.
+* Tap the "ON/OFF" button while continuing to hold the "VOL-" button for ~5 seconds
```

Comment:
i don't think we need both Vol - and pressing on/off button. i checked with @andersson and he confirmed. if you agree and fix that, then i think we can merge the rest of the changes.

Reply:
I think needing to tap the on/off button depends on the state of the usb-c cable. In some cases you really do need to tap the on/off, so having it here helps make sure these instructions always work.

Reply:
(Though I do like @sgermanserrano's suggestion to use the blue light as the point where you can release the vol- button, as that is more clear then the ~5 second time span)

Reply:
the current instructions start with "disconnect power and usb", so how does it depend on the state of the usb-c cable?

Reply:
In my experience, if the USB-C cable is plugged in, you don't need to tap the on/off button, it will start booting up without touching anything. However, with the USB-C cable unplugged, powering the system on while holding the vol- button doesn't start the board. You have to tap the ON/OFF button.

Reply:
Isn't that only if pin 3 is ON?

Reply:
@sgermanserrano  Switch 3?  I'm not familiar, do you mean the dip switches? Is that in this documentation?

Reply:
So, can we scope this pull request back down to just the documentation issue of manually describing how to enter fastboot mode?  Can we focus on specific objections to the patch I've submitted?  I'd like to get this closed out.

Reply:
@sgermanserrano / @ndechesne / @ric96  Can we figure out what to do here? I feel like 20 days to come to agreement on a minor documentation update is showing some serious review dysfunction.

I still hold that my description is the most accurate and consistent way to manually enter into fastboot mode. The board has too many variants in how it boots depending on how it might have crashed to leave any of the steps out.

Reply:
@johnstultz-work might be worth adding the bit on the blue LED next to the ~5sec holding time. Apart from that it looks good

Reply:
@johnstultz-work `till the blue led lights up` seems a bit more definitive than `~5 seconds` can you try it on your end and update the pr?
Everything else looks ok.